serverless/tensorflow/mediapipe/nuclio/main.py

```diff
@@ -0,0 +1,80 @@
+import mediapipe as mp
+from mediapipe.tasks import python
+from mediapipe.tasks.python import vision
+import json
+import base64
+import io
+from PIL import Image
+import yaml
+import numpy as np
+def init_context(context):
+    context.logger.info("Init detector...")
+
+    base_options = python.BaseOptions(
+    model_asset_path='pose_landmarker_heavy.task'
+    )
+    options = vision.PoseLandmarkerOptions(
+        base_options=base_options,
+        num_poses = 2,
```

Comment:
Why did you strict number of poses by 2?

Reply:
i think 0 is unlimited , but i cant find mention about it in docs
Should i set this parameter = 1 in this version?

Reply:
I do not think we want to strict this number somehow